spec/features/event_spec.rb

```diff
@@ -135,12 +135,12 @@
       end
     end
 
-    it "should not allow dates in the past" do
+    it "should allow dates in the past" do
```

Comment:
I am not sure if this test communicates the right thing. If I were a new developer to this project I would ask what's the purpose of creating events in the past. Allowing old events to edit makes sense, but creating "old" events?